### PR TITLE
Ignore vulnerabilities feature for CLI

### PR DESCRIPTION
**Pull request by AkshayBhansali18**
Ignore Vulnerability Feature for Maven, pypi, golang, and npm ecosystems.

### CODE REVIEW

File: pkg/analyses/summary/types.go [hunk 1]
```diff
@@ -29,5 +32,7 @@ type ProcessVulnerabilities struct {
 	PrivateVul                   int
 	DirectVulnerableDependencies int
 	TotalTransitives             int
+	TotalDirectIgnored			 int
+	TotalTransitiveIgnored		 int
```
Comment (sawood14012) [inline pkg/analyses/summary/types.go:36]: same here

**Review by sawood14012 — CHANGES_REQUESTED** [review]

**Comment by sawood14012** [comment]
why does package.json has the key ignore but all other ecosystems has crdaignore ?

**Comment by AkshayBhansali18** [comment]
> why does package.json has the key ignore but all other ecosystems has crdaignore ?

@sawood14012 because a json file doesn't allow comments, but comments are allowed in manifests of other ecosystems.

**Comment by sawood14012** [comment]
> > why does package.json has the key ignore but all other ecosystems has crdaignore ?
> 
> @sawood14012 because a json file doesn't allow comments, but comments are allowed in manifests of other ecosystems.

no i was just taking about the key of the json object

**Comment by AkshayBhansali18** [comment]
> > > why does package.json has the key ignore but all other ecosystems has crdaignore ?
> > 
> > 
> > @sawood14012 because a json file doesn't allow comments, but comments are allowed in manifests of other ecosystems.
> 
> no i was just taking about the key of the json object

Ohh, thats because the json sent to the server is sent with key ignore. It could be changed to crdaignore too.

**Comment by sawood14012** [comment]
> > > > why does package.json has the key ignore but all other ecosystems has crdaignore ?
> > > 
> > > 
> > > @sawood14012 because a json file doesn't allow comments, but comments are allowed in manifests of other ecosystems.
> > 
> > 
> > no i was just taking about the key of the json object
> 
> Ohh, thats because the json sent to the server is sent with key ignore. It could be changed to crdaignore too.

yup please do

**Review by sawood14012 — APPROVED** [review]
LGTM

**Pull request merged**